docs/user/azure/install_upi.md

```diff
@@ -342,7 +342,8 @@ Copy the [`04_bootstrap.json`](../../../upi/azure/04_bootstrap.json) ARM templat
 Create the deployment using the `az` client:
 
 ```sh
-export BOOTSTRAP_URL=`az storage blob url --account-name ${CLUSTER_NAME}sa --account-key $ACCOUNT_KEY -c "files" -n "bootstrap.ign" -o tsv`
+export end=`date -u -d "10 hours" '+%Y-%m-%dT%H:%MZ'`
```

Comment:
This does not need to b exported. I would also recommend a more specific name than just "end".
```suggestion
bootstrap_url_expiry=`date -u -d "10 hours" '+%Y-%m-%dT%H:%MZ'`
```